### ISSUE TITLE
Remove jQuery dependency

### ISSUE THREAD

**Issue by NickSto** [galaxyproject/galaxy-hub#785]
jQuery still exists as a dependency because certain Bootstrap plugins require it:
- the [modal](https://getbootstrap.com/docs/4.6/components/modal/) vimeo embed in [/tutorials/chip/](https://galaxyproject.org/tutorials/chip/#data-location)
- the collapsible sections in [/tutorials/nt-rnaseq/](https://galaxyproject.org/tutorials/nt-rnaseq/#how-to-upload-the-data-into-galaxy)
- probably other special elements in other content pages

Also, the [tabbed](https://getbootstrap.com/docs/4.6/components/navs/#tabs) panes in [/use/](https://galaxyproject.org/use/) and [/search/](https://galaxyproject.org/search/) used to depend on Bootstrap, but have been replaced with BootstrapVue components.

325cefee1 enabled these plugins by adding jQuery, but this failed on build. So 98f1bdfd1 and 6a1ec9795 worked around this by manually including the Bootstrap and jQuery files in `<script>` tags.

This jQuery dependency is not excellent, since we want to avoid using jQuery in our scripts and instead use Vue and its capabilities. Also, we only need it for these few instances where we're using plugins from Bootstrap 4. Version 5 has actually done away with the jQuery dependency.

So we'd like to eventually drop the jQuery dependency entirely by one or both of the following routes:
1. Replace all instances of elements requiring jQuery-dependent Bootstrap plugins with Vue components.
2. Upgrade to Bootstrap 5.

**Comment by NickSto** [comment]
## Pages that might use Bootstrap stuff
Based on the elements in the tutorials pages above, I'm searching for similar content in the rest of the pages.

### modal vimeo embed in /tutorials/chip/
```
$ git grep -lE 'class=[^>]*modal'
content/tutorials/chip/index.md
```
```
$ git grep -E 'class=[^>]*embed'
content/tutorials/chip/index.md:        <div class="embed-responsive embed-responsive-16by9">
content/ftp-upload/index.md:<div class="embed-responsive embed-responsive-16by9">
content/tutorials/collections/index.md:<div class="embed-responsive embed-responsive-16by9">
content/tutorials/ngs/index.md:<div class="embed-responsive embed-responsive-16by9">
content/tutorials/rb-rnaseq/index.md:> <div class="embed-responsive embed-responsive-16by9">
content/tutorials/upload/index.md:<div class="embed-responsive embed-responsive-16by9">
```

### collapsible sections in /tutorials/nt-rnaseq/
```
$ git grep -lE 'class=[^>]*panel'
content/tutorials/nt-rnaseq/index.md
```
```
$ git grep -lE '<[^>]+data-toggle'
content/tutorials/chip/index.md
content/tutorials/nt-rnaseq/index.md
```

**Comment by NickSto** [comment]
Closed with #1105.

I checked the video embeds I found with the `class=[^>]*embed` search above and they still work fine.
